drools-wb-services/drools-wb-verifier/drools-wb-verifier-backend/src/main/java/org/drools/workbench/services/verifier/plugin/backend/VerifierWebWorkerServlet.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright 2016 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.drools.workbench.services.verifier.plugin.backend;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.Reader;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static java.lang.String.*;
+
+public class VerifierWebWorkerServlet
+        extends HttpServlet {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger( VerifierWebWorkerServlet.class );
+
+    public static String loadResource( final String name ) throws
+                                                           Exception {
+        final InputStream in = VerifierWebWorkerServlet.class.getResourceAsStream( name );
+        final Reader reader = new InputStreamReader( in );
```

Comment:
I think it's customary to add the encoding; UTF-8.. IIRC @psiroky went though the codebase a while ago adding missing encoding... might be worth checking.